core/src/main/java/brooklyn/location/access/PortForwardManager.java

```diff
@@ -46,6 +47,42 @@
 @Beta
 public interface PortForwardManager extends Location {
 
+    @Beta
+    class AssociationMetadata {
```

Comment:
Worth adding `toString()` to this - you rely on a nice toString in a debug message of https://github.com/brooklyncentral/advanced-networking/pull/39

e.g.:

    public String toString() {
        return Objects.toStringHelper()
                .add("publicIpId", publicIpId)
                .add("publicEndpoint", publicEndpoint)
                .add("location", location)
                .add("privatePort", privatePort)
                .build();
    }